src/dataprotection/README.md

```diff
@@ -0,0 +1,201 @@
+    --resource-group "000pikumar" --vault-name "PratikPrivatePreviewVault1" 
+```
+##### Delete #####
+```
+az dataprotection backup-instance delete --name "testInstance1" --resource-group "000pikumar" \
+    --vault-name "PratikPrivatePreviewVault1" 
+```
+#### dataprotection recovery-point ####
+##### List #####
+```
+az dataprotection recovery-point list --backup-instance-name "testInstance1" --resource-group "000pikumar" \
+    --vault-name "PratikPrivatePreviewVault1" 
+```
+##### Show #####
+```
+az dataprotection recovery-point show --backup-instance-name "testInstance1" \
+    --recovery-point-id "7fb2cddd-c5b3-44f6-a0d9-db3c4f9d5f25" --resource-group "000pikumar" \
+    --vault-name "PratikPrivatePreviewVault1" 
+```
+#### dataprotection job ####
+##### List #####
+```
+az dataprotection job list --resource-group "BugBash1" --vault-name "BugBashVaultForCCYv11"
+```
+##### Show #####
+```
+az dataprotection job show --job-id "3c60cb49-63e8-4b21-b9bd-26277b3fdfae" --resource-group "BugBash1" \
+    --vault-name "BugBashVaultForCCYv11" 
+```
+#### dataprotection restorable-time-range ####
+##### Find #####
+```
+az dataprotection restorable-time-range find --backup-instances "zblobbackuptestsa58" \
+    --end-time "2021-02-24T00:35:17.6829685Z" --source-data-store-type "OperationalStore" \
+    --start-time "2020-10-17T23:28:17.6829685Z" --resource-group "Blob-Backup" --vault-name "ZBlobBackupVaultBVTD3" 
+```
+#### dataprotection export-job ####
+##### Trigger #####
+```
+az dataprotection export-job trigger --resource-group "SwaggerTestRg" --vault-name "NetSDKTestRsVault"
+```
+#### dataprotection export-job-operation-result ####
+##### Show #####
+```
+az dataprotection export-job-operation-result show --operation-id "00000000-0000-0000-0000-000000000000" \
+    --resource-group "SwaggerTestRg" --vault-name "NetSDKTestRsVault" 
```

Comment:
`az dataprotection export-job show-operation-result` is better.

Reply:
Hi @kairu-ms this file seems to be autogenerated and not hand written. And it's not exactly picking from the available list of commands. We have actually taken these feedback earlier and updated our readme.az.md to rename the cmdlets

Reply:
Can you show me the  readme.az.md configuration. Let me check.

Reply:
https://github.com/sambitratha/azure-rest-api-specs/blob/dpp-private/specification/dataprotection/resource-manager/readme.az.md

Reply:
Can you add the following configuration under `directive` to checkout whether rename works.

```
  - where:
      command: dataprotection export-job-operation-result show
    set:
      command: dataprotection export-job-operation-result show-operation-result
  - where:
      group: dataprotection export-job-operation-result
    set:
      group: dataprotection export-job
```